src/components/project/ProjectList.tsx

```diff
@@ -21,134 +20,158 @@ export default class ProjectList extends Component<ProjectListProps, ProjectList
                 name: true,
             },
             errorMessage: {
-                name: ""
-            }
+                name: '',
+            },
         }
-        this.saveProject = this.saveProject.bind(this);
-        this.handleChange = this.handleChange.bind(this);
-        this.discard = this.discard.bind(this);
-        this.addProject = this.addProject.bind(this);
+        this.saveProject = this.saveProject.bind(this)
+        this.handleChange = this.handleChange.bind(this)
+        this.discard = this.discard.bind(this)
+        this.addProject = this.addProject.bind(this)
     }
 
- getProjectList = () => {
-    getProjectList().then((response) => {
-        this.setState({
-            view: ViewType.FORM,
-            code: response.code,
-            projects: response.result.reverse()
-        })
-    }).catch((errors) => {
-        if (Array.isArray(errors.error)) {
-            errors.error.map((err) => toast.error(err.userMessage));
-            this.setState({ view: ViewType.ERROR, code: errors.code, loadingData: false })
-        }
-    })
- }
+    getProjectList = () => {
+        getProjectList()
+            .then((response) => {
+                this.setState({
+                    view: ViewType.FORM,
+                    code: response.code,
+                    projects: response.result.reverse(),
+                })
+            })
+            .catch((errors) => {
+                if (Array.isArray(errors.error)) {
```

Comment:
Can use showError instead

Reply:
due to wider scope of testing , leaving for now